src/k8s_api_endpoints.py

```diff
@@ -30,6 +30,18 @@ def from_config(self) -> Optional[str]:
                 if addresses:
                     return build_url(addresses[0], 6443)
 
+    def get_external_api_endpoints(self) -> List[str]:
+        response = self.charm.lb_external.get_response("api-server-external")
+        if not response or response.error:
+            return []
+        return [response.address]
+
+    def get_internal_api_endpoints(self) -> List[str]:
+        response = self.charm.lb_external.get_response("api-server-internal")
+        if not response or response.error:
+            return []
+        return [response.address]
```

Comment:
Is it necessary for it to be a list containing only one element? This looks odd...